library/spark/spark-accumulo-library/src/test/java/uk/gov/gchq/gaffer/sparkaccumulo/operation/handler/scalardd/GetRDDOfElementsHandlerTest.java

```diff
@@ -377,4 +377,22 @@ public void checkGetCorrectElementsInRDDForEdgeId() throws OperationException, I
         sparkContext.stop();
     }
 
+    @Test(expected = java.lang.IllegalArgumentException.class)
+    public void testNoSparkContext() throws OperationException {
+        final Graph graph1 = new Graph.Builder()
+                .addSchema(getClass().getResourceAsStream("/schema/dataSchema.json"))
```

Comment:
As discussed, leaving this for now as the schema folder contains several different files when only a subset are required.